app/models/rook.rb

```diff
@@ -1,2 +1,8 @@
 class Rook < Piece
+  def valid_move?(to_row, to_col)
+    # Returns to false if pathis obstructed, as Rook cannot jump its own pieces
+    # return false if piece.path_obstructed?
+    # Vertical move col stays the same # Horizontal move row stays the same
+    return true if (col - to_col).zero? || (row - to_row).zero?
```

Comment:
Technically, this would work because `move_nil?` is called before `valid_move`. If this was not the case, your conditional would have to look something like `(col is same AND row is different) OR (col is ...`

Also, it's shorter if you just use `col == to_col`